hello.gno

```diff
@@ -0,0 +1,31 @@
+package hello
+
+// message is a global variable that stores the message to be displayed.
+var message string = "Hello, Gno!"
+
+// Render is used by gnoweb to display dynamic content.
+func Render(path string) string {
+	return `# ` + message + `
+
+This is a simple Gno realm that demonstrates basic functionality.
+Try it out with gnokey:
+
+	gnokey add myaccount
+	gnokey list
```

Comment:
Since the user just added this key to `gnokey`, `gnodev` will have to be restarted to avoid the error message `account XXX does not exist`

Reply:
@gfanton is there a shortcut or something we can use to make it happening?

(could we consider doing it automatically?)